### ISSUE TITLE
Heroic button initial setup

### ISSUE THREAD

**Issue by alistairstead** [gitevents/core#148]
What would you think of an updated README.md paragraph on getting started that includes a [heroku button](https://blog.heroku.com/archives/2014/8/7/heroku-button)? This would allow the creation of a single click setup and wizard to prompt for the required config.

It would require a minor config to the config file to allow overloading using ENV vars to make the app a little more 12 factor.

**Comment by PatrickHeneise** [comment]
Heroku was a previous idea https://github.com/gitevents/core/issues/56

I discarded the idea since the free heroku instances aren't always on and a webhook needs a direct answer or times out. I think we can have the heroku install as an alternative to the Digital Ocean version (which is $2/M cheaper).

Other opinions?

**Comment by mike182uk** [comment]
I dont think this is a bad idea actually, in fact i would say it could be a better option to replace the whole digital ocean setup with the heroku button:

1. You would have a 1 click installer, not a "Almost-just-one-click-ready-to-launch" installer ;)
2. Its a far lower barrier to entry to setup on heroku than it is digital ocean (user clicks a button, clones project and modifies to suit)
3. Whilst digital ocean is a little cheaper, heroku's free tier is still valuable for people who may just want to checkout how the system works to see if it would be a good fit for there user group. Its also easy to scale to a paid tier if the user decides to do so
4. We can reduce the amount of steps the user needs to take to get up and running and remove some cruft from the readme
5. Should the core repo even discuss how to get a user setup on Digital Ocean? What about other providers? Would we not be better just providing instructions on how to get setup without specifying a platform and let the user tailor to there specific platform?
6. We could get rid the `dockerfile` and `cloudconfig.yml` to avoid any unnecessary confusion
